clients/client-web/package.json

```diff
@@ -10,7 +10,7 @@
   ],
   "scripts": {
     "lint": "eslint src/*.js test/*.js",
-    "test": "karma start --single-run"
+    "test": "export NODE_OPTIONS=--openssl-legacy-provider && karma start --single-run"
```

Comment:
Since Node.js v17, it now [supports OpenSSL v3.0](https://nodejs.org/en/blog/release/v17.0.0/#openssl-3-0), which brings tightened restrictions on the allowed algorithms and key sizes. Because we can't upgrade webpack to the latest version (Neutrino doesn't support), we need to use the legacy provider to avoid the ERR_OSSL_EVP_UNSUPPORTED error.